frontend/src/v4/routes/viewerGui/components/gis/gis.styles.ts

```diff
@@ -92,6 +95,4 @@ export const MapName = styled.div`
 	margin-left: 24px;
```

Comment:
I'm not exactly sure what's caused this but there is a horizontal scrollbar now appearing in the v4 GIS window